test/test-utils.js

```diff
@@ -43,3 +44,23 @@ test('joinListConjuction test', function (t) {
   t.equal(utils.joinListConjuction(things, ', ', ' and '), 'this, that and other');
   t.equal(utils.joinListConjuction(thing, ', ', ' and '), 'this');
 });
+
+test('makeGetStationResponseText tests', function (t) {
```

Comment:
Can we get a bit more descriptive with the name of this test plan? Maybe `'test makeGetStationresponseText results for zero, single, or multiple stations'`